ReferenceImplementation/ReferenceProject.backend/src/ReferenceProject/backend/entities/models/Complaint.java

```diff
@@ -44,6 +47,10 @@
 	@XmlElement(nillable=true)
 	protected ComplaintStatus status;
 	
+	@OneToOne
+	@XmlElement(nillable=true)
+	protected WorkflowState workflowState;
```

Comment:
Removed reference temporarily since it belongs to the task of connecting the data with the workflow state and has nothing to do with the list of open issues